js/plugins/googleai/tests/gemini_test.ts

```diff
@@ -463,7 +463,7 @@ describe('plugin', () => {
       assert.strictEqual(flash.__action.name, 'googleai/gemini-1.5-flash');
     });
 
-    it('references explicitly registered models', async () => {
+    it('references both pre-registered or dynamic models', async () => {
```

Comment:
maybe a separate test case for dynamic?

Reply:
I can separate them in a new PR since you want this one merged ASAP